Tests.Integration.Presentation.Web/UI_Configuration/UIConfigurationTests.cs

```diff
@@ -51,11 +47,103 @@ public async Task Can_Get()
+            var (cookieUser2, _) = await CreatePrerequisitesAsync();
+
+            await UIConfigurationHelper.CreateUIModuleAndSaveAsync(organization.Id, module, cookie);
 
-            var data = await response.ReadResponseBodyAsKitosApiResponseAsync<UIModuleCustomizationDTO>();
-            Assert.Empty(data.Nodes);
-            Assert.NotNull(data.Nodes);
+            //Act
+            using var response = await UIConfigurationHelper.SendGetRequestAsync(organization.Id, module, cookieUser2);
+
+            //Assert
+            Assert.Equal(HttpStatusCode.Forbidden ,response.StatusCode);
+        }
+
+        [Fact]
+        public async Task Put_Deletes_Nodes()
+        {
+            var module = A<string>();
+            var multipleUiCustomizations= UIConfigurationHelper.PrepareTestUiModuleCustomizationDto(10);
+            var singleUiCustomization = UIConfigurationHelper.PrepareTestUiModuleCustomizationDto();
+            
+            using var firstPutResponse = await UIConfigurationHelper.SendPutRequestAsync(TestEnvironment.DefaultOrganizationId, module, multipleUiCustomizations);
+            Assert.Equal(HttpStatusCode.NoContent, firstPutResponse.StatusCode);
+
+            using var secondPutResponse = await UIConfigurationHelper.SendPutRequestAsync(TestEnvironment.DefaultOrganizationId, module, singleUiCustomization);
+            Assert.Equal(HttpStatusCode.NoContent, secondPutResponse.StatusCode);
+
+            var response = await UIConfigurationHelper.GetCustomizationByModuleAsync(TestEnvironment.DefaultOrganizationId, module);
+            Assert.Equal(singleUiCustomization.Nodes.Count(), response.Nodes.Count());
+        }
+
+        [Fact]
+        public async Task Put_Returns_BadRequest_If_Keys_Are_Invalid()
+        {
+            //Arrange
+            var module = A<string>();
+            var incorrectKey = "Incorrect_Key123";
+            var uiModuleCustomizationDto = UIConfigurationHelper.PrepareTestUiModuleCustomizationDto(1, incorrectKey);
+
+            //Act
+            using var putResponse = await UIConfigurationHelper.SendPutRequestAsync(TestEnvironment.DefaultOrganizationId, module, uiModuleCustomizationDto);
+
+            //Assert
+            Assert.Equal(HttpStatusCode.BadRequest, putResponse.StatusCode);
+        }
+
+        [Fact]
+        public async Task Put_Returns_NotFound_If_Organization_Doesnt_Exist()
+        {
+            //Arrange
+            var module = A<string>();
+            var incorrectOrgId = -1;
+            var uiModuleCustomizationDto = UIConfigurationHelper.PrepareTestUiModuleCustomizationDto();
+
+            //Act
+            using var putResponse = await UIConfigurationHelper.SendPutRequestAsync(incorrectOrgId, module, uiModuleCustomizationDto);
+
+            //Assert
+            Assert.Equal(HttpStatusCode.NotFound, putResponse.StatusCode);
+        }
+
+        [Fact]
+        public async Task Put_Returns_Forbidden_If_Unauthorized()
```

Comment:
Perhaps be more helpful. `Forbidden_If_User_Is_Not_Local_Admin_In_The_Target_Organization`

Also consider not just using a cookie from another org but actually using a cookie from the same org, but where the user is not local admin